distributed/deploy/utils_test.py

```diff
@@ -25,10 +25,14 @@ def test_submit(self):
         assert future.result() == 2
 
     def test_context_manager(self):
-        with self.Cluster(**self.kwargs) as c:
+        kwargs = self.kwargs.copy()
+        kwargs.pop("dashboard_address")
```

Comment:
This would silently ignore any `dashboard_address` which has been specified in a test. Since this is only ever used in one place, perhaps we should just specify `dashboard_address=":0"` there instead? This isn't worth spending too much time on, it was just slightly surprising to see us manually setting `dashboard_address=":54321"`

Reply:
yes, I think I changed this before I thought about the idea to simply use a random port which ignores the warning. I'll fix that